bundles/core/org.eclipse.smarthome.core.thing.test/src/test/groovy/org/eclipse/smarthome/core/thing/firmware/FirmwareRegistryOSGiTest.groovy

```diff
@@ -325,29 +324,29 @@ final class FirmwareRegistryOSGiTest extends OSGiTest {
     }
 
     @Test(expected=UnsupportedOperationException)
-    void 'assert that firmware properties are immutable'() {
-        def fw = firmwareRegistry.getFirmware(FW112_EN.getUID())
+    void 'assert that firmware properties are immutable'(){
+        def fw = firmwareRegistry.getFirmware(thing1, FW112_EN.getVersion())
         assertThat fw, is(notNullValue())
         fw.getProperties().put("test", null)
     }
 
     @Test
-    void 'void assert that registry returns null for unknown thing type uid and getLatestFirmware operation()'() {
-        def firmware = firmwareRegistry.getLatestFirmware(UNKNOWN_THING_TYPE_UID);
+    void'void assert that registry returns null for unknown thing type uid and getLatestFirmware operation'() {
```

Comment:
I think we can remove the void term in the operation name and add an additional blank between ```void 'assert...```